src/Sparrow/Logging/Logger.cs

```diff
@@ -13,6 +13,13 @@ public class Logger
         [ThreadStatic]
         private static LogEntry _logEntry;
 
+        private static readonly long LocalToUtcOffsetInTicks;
+
+        static Logger()
+        {
+            LocalToUtcOffsetInTicks = TimeZoneInfo.Local.GetUtcOffset(DateTime.Now).Ticks;
```

Comment:
Might need to update it every now and then for things like daylight savings changes?

Reply:
Can probably get away with doing that only once every file, or when we are about to go idle?

Reply:
I'll fix that in a bit